extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -123,6 +123,8 @@ export class AmpSidebar extends AMP.BaseElement {
     // Get the toolbar attribute from the child navs.
     const toolbarElements = toArray(element.querySelectorAll('nav[toolbar]'));
 
+    // Add found toolbar elements, as new Toolbars.
```

Comment:
nit: Instead of a period, could we do a comma?

Reply:
Good point. Could we do a semi colon?

Reply:
Semi Colons are cool. But I think period would be best here.

Reply:
Done.